rpi-update

```diff
@@ -115,10 +115,39 @@ function update_sdk {
 }
 
 function show_notice {
-	local NOTICE=`curl -Lfs https://raw.githubusercontent.com/hexxeh/rpi-firmware/${FW_REV}/NOTICE.md`
+	local FULL_NOTICE=`curl -Lfs https://raw.githubusercontent.com/hexxeh/rpi-firmware/${FW_REV}/NOTICE.md`
+	local NOTICE=$(echo "$FULL_NOTICE" | tail -n+2)
 	if [ -z "$NOTICE" ]; then
 		return
 	fi
+	local NOTICE_HASH_HEAD=$(echo "$FULL_NOTICE" | head -1)
+	if [ $(echo "${NOTICE_HASH_HEAD}" | awk -F: '{print $1}') == "HASH" ]; then
+		local NOTICE_HASH_EXISTS=true
```

Comment:
For style-consistency with the rest of the code in this script, you should probably use `1` and `0` instead of `true` and `false`, and then change all your conditionals to e.g. `if [[ ${NOTICE_HASH_EXISTS} -eq 1 ]]; then`